assets/collection-filters-form.js

```diff
@@ -77,10 +82,13 @@ class CollectionFiltersForm extends HTMLElement {
 
   renderProductCount(html) {
     const count = new DOMParser().parseFromString(html, 'text/html').getElementById('CollectionProductCount').innerHTML
+    const container = document.getElementById('CollectionProductCount');
```

Comment:
For most of those `querySelector`s and `getElementById` wouldn't it be better to query `this` and limit it to the custom element we're dealing with ?

Reply:
In this case, `#CollectionProductCount` is not a descendent of `<collection-filters-form>` custom element